src/lib/storage/segment_accessor.hpp

```diff
@@ -71,7 +79,7 @@ class MultipleChunkReferenceSegmentAccessor final : public AbstractSegmentAccess
 
     const auto chunk_id = row_id.chunk_id;
 
-    // Grow the _accessors vector faster than linearly if the chunk_id is out of its current bounds
+    // Grow the _accessor's vector faster than linear if the chunk_id is out of its current bounds
```

Comment:
Both look correct to me in the original version

Reply:
I am not sure about "linearly". I would say it refers to to "faster" and not to "grow", but I really have no idea.

Reply:
It does. In any case, "faster" is an comparative adjective, "grow" is a verb. Both are described by the adverb form (linearly).